chromium/pages/options/ux.js

```diff
@@ -331,27 +331,19 @@ document.addEventListener("DOMContentLoaded", () => {
   const disabled_site_input = document.getElementById("disabled-site");
   const add_disabled_site_invalid_host = document.getElementById('add-disabled-site-invalid-host');
   disabled_site_input.setAttribute("placeholder", chrome.i18n.getMessage("options_enterDisabledSite"));
-  function isValidHost(host) {
-    try {
-      new URL(`http://${host}/`);
-      return true;
-    } catch {
```

Comment:
It's possible to fix the lint error without removing this code completely, just replace this line with:

```
    } catch (_) {
```

Reply:
It seems that the URL constructor throw TypeError only. And it will not throw error for some common invalid URLs.  

```node
> try { new URL('https://*...,example.com/'); console.log('ok'); } catch(error) { console.log(error); }
ok

> try { new URL(123); console.log('ok'); } catch(error) { console.log(error); }
TypeError [ERR_INVALID_URL]: Invalid URL: 123
    at onParseError (internal/url.js:256:9)
    at new URL (internal/url.js:332:5)
    at repl:1:7
    at Script.runInThisContext (vm.js:120:18)
    at REPLServer.defaultEval (repl.js:433:29)
    at bound (domain.js:427:14)
    at REPLServer.runBound [as eval] (domain.js:440:12)
    at REPLServer.onLine (repl.js:760:10)
    at REPLServer.emit (events.js:327:22)
    at REPLServer.EventEmitter.emit (domain.js:483:12) {
  input: '123',
  code: 'ERR_INVALID_URL'
}
undefined
```

The `disable_on_site` will validate the hostname and returns an error by default. IMHO we should rely on a single-source-of-truth implementation for validations.

https://github.com/EFForg/https-everywhere/blob/2bfa7efe3575f8b38827f5eb278d47ff14608095/chromium/background-scripts/background.js#L899-L907

Reply:
Seems like a good idea indeed.